packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -222,9 +228,9 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
     delete _allInstances[this._id];
 
     if (!this._isInnerZone) {
-      _outerZones.delete(this);
+      _outerZones.get(this.windowElement!)?.delete(this);
```

Comment:
These should be an additional check to remove entries from `_outerZones`

Reply:
Which check?